### PR TITLE
DEV-14582 - Use common security group module

### PR DESCRIPTION
**Pull request by souza-dan**
https://datatamr.atlassian.net/browse/DEV-14582

### CODE REVIEW

File: modules/aws-security-groups/variables.tf [hunk 1]
```diff
@@ -1,106 +1,20 @@
...
+  default = [
+    22,
+    9100,  // UI
+    9200,  // ES
+    9020,  // auth
+    9080,  // persistence
+    21281, // zookeeper
+    5601,  // kibana
+    31101, // grafana
+  ]
```
Comment (schwartzmanb) [inline modules/aws-security-groups/variables.tf:13]: Any need for 80 or 443? or just leave those for users to add if needed?

**Review by schwartzmanb — APPROVED** [review]
LGTM

File: modules/aws-security-groups/variables.tf (hunk 1 above)
Comment (souza-dan) [reply modules/aws-security-groups/variables.tf:13]: You might need those if you installed nginx and configured it to use those ports. Seems like a good use for `additional_ports` variable.

**Review by daniosim — APPROVED** [review]
Looks good! 👍

**Review by aditimarkale — APPROVED** [review]

**Pull request merged**